docs/guide/writing/style-guides.md

```diff
@@ -49,34 +51,104 @@ If you belong to an open-source community or NGO, you can consider the following
 
 Technical documentation takes many forms, each one targeting a certain need. Here, we can think of examples like tutorials, API documentation, and user manuals. So while many style guides adequately cover the main concerns of all technical documentation, sometimes a more specialized guide is needed.
 
-### Thinking about accessibility
+## Style for developers
+
+Developer documentation often have a specific set of rules in order to best meet their needs.
```

Comment:
Personally, I'd use `has` here, and I find the `their` slightly confusing.

Reply:
Absolutely. This has been addressed.